### PR TITLE
Add discussion and note on file attrs.

### LINKED ISSUE #6305: FAQ: why borg (esp. 1st backup) is slower than rsync
See my post in https://github.com/borgbackup/borg/discussions/6304 .

With a little editing, this could be put into the borg docs, FAQ section.

### PR DESCRIPTION
**Pull request by m3nu**
Extend FAQ by adding answer from https://github.com/borgbackup/borg/discussions/6304 and note on file attrs.

Fixes #6305

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -880,13 +880,30 @@ If you run into that, try this:
...
+  if your CPU has sha256 acceleration) or blake2b (fast, in software))
+- Then it checks whether this chunk is already in the repo (local hashtable lookup, 
+  fast). If so, the processing of the chunk is completed here. Otherwise it needs to 
+  process the chunk:
+- Compresses the chunks (the default lz4 is super fast)
+- Encrypts (AES, usually fast if your CPU has AES acceleration as usual
+  since about 10y)
+- Authenticates ("signs") using hmac-sha256 or blake2b (see above),
+- Transmits to repo (remote repo), usually involving an SSH connection 
+  (does its own encryption)
```
Comment (ThomasWaldmann) [inline docs/faq.rst:898]: Transmits to repo. If the repo is remote, this usually involves an SSH connection (does its own encryption / authentication).

File: docs/faq.rst [hunk 2]
```diff
@@ -880,13 +880,30 @@ If you run into that, try this:
...
+This can make creation of the first archive slower, but saves time 
+and disk space on subsequent runs. Here what Borg does when you run ``borg create``:
+
+- Borg chunks the file (using the relatively expensive buzhash algorithm)
+- It then computes the "id" of the chunk (hmac-sha256 (often slow, except 
+  if your CPU has sha256 acceleration) or blake2b (fast, in software))
+- Then it checks whether this chunk is already in the repo (local hashtable lookup, 
+  fast). If so, the processing of the chunk is completed here. Otherwise it needs to 
+  process the chunk:
+- Compresses the chunks (the default lz4 is super fast)
```
Comment (ThomasWaldmann) [inline docs/faq.rst:893]: Compresses (...)

Not "the chunks" because we are only talking about 1 chunk here, see previous item.

**Review by ThomasWaldmann — COMMENTED** [review]
thanks for the pr!

**Comment by m3nu** [comment]
Thanks for reviewing! Updated based on feedback.

**Pull request merged**